test/unit/calculators/benefit_cap_calculator_data_query_test.rb

```diff
@@ -4,8 +4,8 @@ module SmartAnswer::Calculators
   class BenefitCapCalculatorDataQueryTest < ActiveSupport::TestCase
     context BenefitCapCalculatorDataQuery do
       setup do
-        query = BenefitCapCalculatorDataQuery.new
-        @rates = query.data.fetch("rates")
+        @query = BenefitCapCalculatorDataQuery.new
+        @rates = @query.data.fetch("rates")
```

Comment:
This commit is essentially empty. Are there other changes that should be included?